src/components/coin-row/FastTransactionStatusBadge.tsx

```diff
@@ -25,20 +25,27 @@ export default React.memo(function FastTransactionStatusBadge({
   style?: StyleProp<ViewStyle>;
 }) {
   let statusColor = useForegroundColor('labelTertiary');
-  if (transaction?.status === 'pending') {
+  if (transaction?.status === TransactionStatus.pending) {
     statusColor = colors.appleBlue;
   } else if (transaction?.status === 'failed') {
     statusColor = colors.red;
   }
 
+  const getStatusTitle = useCallback((status: TransactionStatus, title: string) => {
+    const transactionType = lang.l.transactions.type[title as keyof typeof lang.l.transactions.type];
+    if (typeof transactionType === 'string') {
+      return transactionType;
+    }
+    return transactionType[status as keyof typeof transactionType] || '';
+  }, []);
+
```

Comment:
I don't know if this was intentional or not, but our i18n for some keys had `pending | failed | confirmed` keys inside of `lang.l.transactions.type`. I need to test this to confirm.